src/components/styles/Calendar.styled.js

```diff
@@ -0,0 +1,176 @@
+import styled, { css } from 'styled-components';
+
+const marginHorizontal = css`
+  margin: 0 5px;
+`;
+
+export const CalendarWrapper = styled('div')`
```

Comment:
NABD: Here, you could use the new syntax styled.div``;

Reply:
Done